runners/cromwell_on_google/wdl_runner/test-wdl/basic.papi.us.options.json

```diff
@@ -0,0 +1,6 @@
+{
+  "default_runtime_attributes": {
+    "zones": "us-central1-a us-central1-b us-central1-c us-central1-f us-east1-b us-east1-c us-east1-d us-west1-a us-west1-b" 
```

Comment:
Might want to change this to a specific region, such as us-central1.
Before DRA was deprecated, using any US zone made sense as data was typically in a Standard or DRA US bucket.

With Google's encouragement now of Regional buckets as a replacement for DRA, one should be more careful about which zones they run their compute in. Running Compute in a different region than the user's bucket will generate unnecessary egress charges.

See https://cloud.google.com/compute/pricing#network